### ISSUE TITLE
Repository not up-to-date

### ISSUE THREAD

**Issue by koraytaylan** [meganz/chrome-extension#3]
Extension got an update today with asking new permissions for accessing and editing every website I visit.

I wanted to have a look why it would be necessary and found out that this repository not updated since april and manifest.json shows its version as 1.0.0 whereas in chrome webstore it shows as 3.39.4 and last update as today.

**Comment by Bombasarkadian** [comment]
It looks like the latest release isn't safe and has been injected with malicious code https://twitter.com/serhack_/status/1037026672787304450

**Comment by leo-liar** [comment]
There has just been an new update to 3.39.5 were the malicious code seems to have been removed,

**Comment by serhack** [comment]
@leo-liar Confirmed. 3_39_5_0 is a safe version.

**Comment by leo-liar** [comment]
Yeah, just checked the mega,js myself, the malicious code is gone.

See here for reference:
https://gist.github.com/LI-NA/c096fda8c5d55d38a3a4e501c75bb49e
